docs/source/conf.py

```diff
@@ -48,6 +48,8 @@
 # See: https://nbsphinx.readthedocs.io/en/0.2.15/never-execute.html#Explicitly-Dis-/
 nbsphinx_execute = "never"
 nbsphinx_prolog = """
+|
```

Comment:
why is this needed?

Reply:
it vertically centers the view on github link